lib/brakeman/commandline.rb

```diff
@@ -124,6 +131,18 @@ def regular_report options
         if tracker.options[:exit_on_error] and tracker.errors.any?
           quit Brakeman::Errors_Found_Exit_Code
         end
+
+        if tracker.options[:ensure_ignore_notes]
+          fingerprints = Brakeman::ignore_file_entries_with_empty_notes tracker.ignored_filter&.file
+
+          unless fingerprints.empty?
+            quit Brakeman::Empty_Ignore_Note_Exit_Code,
+                 'Error: notes required for all ignored warnings when ' \
+                 '--ensure-ignore-notes is set. No notes provided for these ' \
+                 'alerts: ' \
+                 "#{fingerprints}"
```

Comment:
Some tweaks here:

I think `[Error]` or `[Notice]` is more inline with Brakeman's messages. Also capitalized `Notes` looks better.

`alerts` should be `warnings` or `fingerprints`

The message is pretty long. Maybe add a newline before the fingerprints? I'd consider one fingerprint per line, too, like the obsolete fingerprints in reports.